data/scripting/editor/editor_controls.lua

```diff
@@ -14,9 +14,7 @@ local r = {
       p(
          -- TRANSLATORS: This is an access key combination. Localize, but do not change the key.
          dl(help_format_hotkey(wl.ui.get_shortcut("encyclopedia")), _("Help")) ..
-         -- TRANSLATORS: This is an access key combination.
-         dl(help_format_hotkey(wl.ui.get_shortcut("editor_menu")), _("Toggle main menu")) ..
-         -- TRANSLATORS: This is an access key combination. The hotkey is 't'
+         -- TRANSLATORS: This is an access key combination. The default hotkey is 't'
```

Comment:
This too. I don't think it's particularly important.

Actually the "access key combination" part should be changed too, maybe to "This is the command for a keyboard shortcut."